ui/src/pages/version/components/forms/components/ResourcesPanel.js

```diff
@@ -106,6 +111,24 @@ export const ResourcesPanel = ({
           />
         </EuiFormRow>
       </EuiForm>
+      <EuiSpacer size="xl" />
+      <EuiCallOut title="Cost Estimation" iconType="tag">
+        <p>
+          The requested resouces will cost between USD{" "}
```

Comment:
typo: should be resources instead of resources

Reply:
Updated